### DISCUSSION TITLE
Is using only the mastercontainer volume from an old backup on a system a problem?

### DISCUSSION THREAD

**Discussion by PhysicsIsAwesome** [nextcloud/all-in-one#5746] [Questions] [answered]
Hi,
I accidentally deleted the mastercontainer volume. Since I thought that backing up only the data dir and database volume is enough, I didn't realize the mastercontainer volume stored important information like database credentials. I only had a backup from quite a few months ago of the mastercontainer volume and restored it from there. Is using this old backup for the mastercontainer volume problematic and if so, what would be a better solution?

**Comment by szaimen** [comment, marked as answer]
> Is using only the mastercontainer volume from an old backup on a system a problem?

No. This should usually work
